fxmanifest.lua

```diff
@@ -1,15 +1,16 @@
 fx_version 'cerulean'
 game 'gta5'
 
-description 'QB-GarbageJob'
+description 'qbx_garbagejob'
 version '1.0.0'
 
 shared_scripts {
-	'@qb-core/shared/locale.lua',
+        '@ox_lib/init.lua',
+        '@qbx_core/import.lua',
```

Comment:
Indentation is still off
![image](https://github.com/Qbox-project/qbx_garbagejob/assets/82737367/1f693479-5470-4e32-a858-9512f32df381)